.github/workflows/build.yaml

```diff
@@ -0,0 +1,100 @@
+name: Check PR
+
+on: pull_request
+
+jobs:
+  # Lint changed templates.
+  xlint:
+    name: Lint templates
+    runs-on: ubuntu-18.04
+
+    env:
+      PATH: '/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin:/usr/local/bin:/tmp/bin'
+      XLINT: '1'
+      LICENSE_LIST: common/travis/license.lst
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 200
+      - name: Fetch upstream
+        run: common/travis/fetch_upstream.sh
+      - name: Get changed templates
+        run: common/travis/changed_templates.sh
+      - name: Fetch xtools
+        run: common/travis/fetch-xtools.sh
+      - name: Run xlint
+        run: common/travis/xlint.sh
+
+  # Build changed packages.
+  build:
+    name: Build packages
+    runs-on: ubuntu-18.04
+    if: "!github.event.pull_request.draft && !contains(github.event.pull_request.title, '[ci skip]' && !contains(github.event.pull_request.body, '[ci skip]')"
```

Comment:
With this `if`, I think the Actions won't build the PR with title or the body (first comment) that has '[ci skip]`, right?

While I like it to not pollute the commit message, I think we should update the README, too.

Reply:
I updated it in CONTRIBUTING, I don't think the README mentions it.